core/modules/taxonomy/taxonomy.module

```diff
@@ -1428,6 +1428,12 @@ function taxonomy_field_formatter_info() {
       'label' => t('RSS category'),
       'field types' => array('taxonomy_term_reference'),
     ),
+    'taxonomy_term_reference_term' => array(
```

Comment:
Will `taxonomy_term_reference_rendered` or `taxonomy_term_reference_entity` be more explicit ?

Reply:
I was using the format that `References` uses (e.g. `user_reference_user`, `node_reference_node`). `Entity Reference` uses `entityreference_entity_view`.

I'm open to any of the approaches but I'll change it to "rendered" since I agree it's a bit more explicit.